### PR TITLE
More fields on IPR report and couple of dropdowns properly populated on impbook

### PR DESCRIPTION
**Pull request by iaincrawf**
IPR report is still missing tests and the entire export function, and impbook is still very much incomplete. This pr will get through the rest of the fields that Yvonne wants on the IPR report for now though

### CODE REVIEW

File: src/Persistence.LinnApps/Repositories/ImportBookRepository.cs [hunk 1]
```diff
@@ -3,6 +3,7 @@
     using System;
     using System.Linq;
     using System.Linq.Expressions;
+    using System.Security.Cryptography.X509Certificates;
```
Comment (richardiphillips) [inline src/Persistence.LinnApps/Repositories/ImportBookRepository.cs:6]: probably not needed

File: src/Service.Host/client/src/components/importBooks/tabs/ImpBookTab.js [hunk 2]
```diff
@@ -417,9 +417,11 @@ function ImpBookTab({
                 </Grid>
 
                 <Grid item xs={6}>
-                    {/* todo implement delivery term code stuff  */}
                     <Dropdown
-                        items={[]}
+                        items={deliveryTerms.map(e => ({
```
Comment (richardiphillips) [inline src/Service.Host/client/src/components/importBooks/tabs/ImpBookTab.js:421]: might be nice to put these items in a variable above

**Review by richardiphillips — APPROVED** [review]

File: src/Service.Host/client/src/components/importBooks/tabs/ImpBookTab.js (hunk 2 above)
Comment (iaincrawf) [reply src/Service.Host/client/src/components/importBooks/tabs/ImpBookTab.js:421]: Do you mean map them somewhere else, like in the container when they're selected? Or to store them in something like a localDeliveryTerms variable? I'd prefer not to store them locally as I've got a few like this and adding consts & useEffects feels a bit cluttered when it's not adding anything

Comment (richardiphillips) [reply src/Service.Host/client/src/components/importBooks/tabs/ImpBookTab.js:421]: i never thought of doing it in the container. That actually might be quite nice if the items aren't used for anything else. I just meant in a const as i think it makes the dropdown declaration a bit cleaner.

Comment (lewisrenfrew) [reply src/Service.Host/client/src/components/importBooks/tabs/ImpBookTab.js:421]: +1 for mapping arrays passed as props into the form actually used by the component in the container

Comment (iaincrawf) [reply src/Service.Host/client/src/components/importBooks/tabs/ImpBookTab.js:421]: Yeah I've got a few where the items aren't used for anything except dropdowns, I'll move the mappings to the container and then can just have nice "items={deliveryTerms}" in the components

**Pull request merged**